features/step_definitions/resources.rb

```diff
@@ -174,8 +174,9 @@
     res = user.cli_exec(:get, **opts)
     case res[:response]
     # the resource has terminated which means we are done waiting.
-    when /Error from server (Forbidden): projects.project.openshift.io/, /not found/, /No resources found/
+    when /Error from server \((Forbidden\)): projects.project.openshift.io/, /not found/, /No resources found/
       break true
```

Comment:
It should escape brackets here, or it will not match always until timeout no matter if the response is expected.

Reply:
There is no need to have extra brackets here. it can be `\(Forbidden\)`